sdk/src/test/scala/com/horizen/api/http/SidechainApiRouteTest.scala

```diff
@@ -327,7 +327,7 @@ abstract class SidechainApiRouteTest extends AnyWordSpec with Matchers with Scal
   val params = MainNetParams(sidechainId = utilMocks.sidechainIdArray)
   val sidechainTransactionApiRoute: Route = SidechainTransactionApiRoute(mockedRESTSettings, mockedSidechainNodeViewHolderRef, mockedSidechainTransactionActorRef,
     sidechainTransactionsCompanion, params).route
-  val sidechainWalletApiRoute: Route = SidechainWalletApiRoute(mockedRESTSettings, mockedSidechainNodeViewHolderRef, sidechainSecretsCompanion).route
+  val sidechainWalletApiRoute: Route = SidechainWalletApiRoute(mockedRESTSettings, mockedSidechainNodeViewHolderRef, null, sidechainSecretsCompanion).route
```

Comment:
Could you please mark variable, what is null?

Reply:
the null variable  is the new SidechainNodeViewReindexer, since it is not used in this test I did not mocked it